packages/frames.js/package.json

```diff
@@ -262,6 +263,7 @@
     "@cloudflare/workers-types": "^4.20240320.1",
     "@types/express": "^4.17.21",
     "@xmtp/frames-validator": "^0.5.2",
+    "@xmtp/proto": "3.45.0",
```

Comment:
eslint will complain because we are importing it in the code but there is no guarantee it is already there, for example if they decide to make it a peer dependency then it can break the build. You should explicitly specify everything you import in `package.json`.

Reply:
ah okay didn't realize we were using it in our code